zeebe/protocol-impl/src/main/java/io/camunda/zeebe/protocol/impl/encoding/AdminRequest.java

```diff
@@ -22,24 +22,29 @@ public class AdminRequest implements BufferReader, BufferWriter {
   private final MessageHeaderDecoder headerDecoder = new MessageHeaderDecoder();
   private final AdminRequestEncoder bodyEncoder = new AdminRequestEncoder();
   private final AdminRequestDecoder bodyDecoder = new AdminRequestDecoder();
-
   private int brokerId = AdminRequestEncoder.brokerIdNullValue();
   private int partitionId = AdminRequestEncoder.partitionIdNullValue();
   private AdminRequestType type = AdminRequestType.NULL_VAL;
-
   private long key = AdminRequestEncoder.keyNullValue();
+  private byte[] configuration = null;
+  private int payloadLength = 0;
+  private boolean hasPayload = false;
```

Comment:
:x: `configuration` should be named `payload`.
:upside_down_face: `payloadLength` can be replaced with `payload.length`
:upside_down_face: `hasPayload` can be replaced with `payload != null`